File: backend/check_db.py
import sqlite3

DB_PATH = r"C:\Users\samruddhi\lumora final\lumora\backend\test.db"
conn = sqlite3.connect(DB_PATH)
conn.row_factory = sqlite3.Row
cur = conn.cursor()

# List all tables
cur.execute("SELECT name FROM sqlite_master WHERE type='table'")
tables = [r[0] for r in cur.fetchall()]
print("Tables:", tables)

if "products" in tables:
    cur.execute("SELECT id, title, price, features, system_requirements, what_you_get, preview FROM products")
    rows = cur.fetchall()
    for r in rows:
        print("---")
        print("ID:", r["id"], "| Title:", r["title"], "| Price:", r["price"])
        print("  features:", r["features"])
        print("  sys_req:", r["system_requirements"])
        print("  what_get:", r["what_you_get"])
        preview = r["preview"] or ""
        print("  preview:", preview[:120])
else:
    print("No products table found!")

conn.close()

File: backend/check_db.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('ecommerce.db');

db.serialize(() => {
  db.each('SELECT id, title, affiliate_enabled FROM products', (err, row) => {
    if (err) console.error(err);
    console.log(`Product: ${row.id} - ${row.title} - AffiliateEnabled: ${row.affiliate_enabled}`);
  });
});

db.close();
